### ISSUE TITLE
No 64bit capable system found. Your hardware is too old.

### ISSUE THREAD

**Issue by johnguild** [liip/php-osx#202]
Please help. 
I was trying to update the php from 5.5 to 5.6 or 7.+
im using MacBook Pro (13 inch, Mid 2012)
with Processor 2.5 Ghz Intel Core i5
and Memory of 4G
-
If i was not mistaken i5 were 64 bit processor, but the one line update 
`curl -s https://php-osx.liip.ch/install.sh | bash -s 7.1`
throws me an error of

Detected OS X El Capitan 10.11. All ok.
****
ERROR! 32 BIT NOT SUPPORTED!
****
No 64bit capable system found. Your hardware is too old.
We don't support that (yet). Patches are welcome ;)
If you think that's wrong, try
****
curl -o install.sh -s https://php-osx.liip.ch/install.sh | bash install.sh force
****

Thanks in advance

**Comment by chregu** [comment]
can you tell me what `sysctl -n hw.cpu64bit_capable` reports?
and `getconf LONG_BIT` ?
Does it work, when you "force" install it?

and i5 should definitely be 64bit capable

**Comment by johnguild** [comment]
getconf LONG_BIT
64
sysctl -n hw.cpu64bit_capable
sysctl: command not found

-
I manage to get the php 5.6 by upgrading to macOS Sierra 10.12 which is the minimun version I needed so far.

Recently I tried this.
curl -s https://php-osx.liip.ch/install.sh | bash -s 7.1
that gives me an error
Detected macOS Sierra 10.12. All ok.
****
ERROR! 32 BIT NOT SUPPORTED!
****
No 64bit capable system found. Your hardware is too old.
We don't support that (yet). Patches are welcome ;)
If you think that's wrong, try
****
curl -o install.sh -s https://php-osx.liip.ch/install.sh | bash install.sh force
****
so I tried again the above code and throws me
PHP 5.6 is not available for OS X < 10.8


As I mentioned above Im able to do what i needed in 5.6, but it would be cool if the update's error could be detected and fix (if there was any). 

-Sorry for my bad english
